dashboard/frontend/main.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable*/
```

Comment:
Oof. Why so drastic?

Reply:
This is my bad, I copied the HTML from my PoC and I had this there. Changed this and fixed the linting errors 😅